docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,688 @@
+{
```

Comment:
<p>Thank you. This is much clearer than the paper!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>